cluster/kubemark/config-default.sh

```diff
@@ -33,7 +33,7 @@ PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 
 MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-debian}
 NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-debian}
-MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-container-vm-v20160321}
```

Comment:
Is this change intentional? This seems to change the GCE's master from CVM to the new image.